### ISSUE TITLE
Getting UndefinedTable error during tab-completion operations in pry

### ISSUE THREAD

**Issue by chrisnicola** [Casecommons/pg_search#223]
For some reason I'm getting this error when I tab-complete on a model function:

```
ClientProfile.where.PG::UndefinedTable: ERROR:  relation "pg_search_documents" does not exist
LINE 5:                WHERE a.attrelid = '"pg_search_documents"'::r...
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod
                FROM pg_attribute a LEFT JOIN pg_attrdef d
                  ON a.attrelid = d.adrelid AND a.attnum = d.adnum
               WHERE a.attrelid = '"pg_search_documents"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum

PG::UndefinedTable: ERROR:  relation "pg_search_documents" does not exist
LINE 5:                WHERE a.attrelid = '"pg_search_documents"'::r...
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod
                FROM pg_attribute a LEFT JOIN pg_attrdef d
                  ON a.attrelid = d.adrelid AND a.attnum = d.adnum
               WHERE a.attrelid = '"pg_search_documents"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum
```

What's odd is that this isn't actually a model that extends PgSearch (though it is related to one) but also that we don't use the multisearch functionality at all, we just use `pg_search_scope` on a single model.

**Comment by nertzy** [comment]
I bet this is because the Active Record model PgSearch::Document is still present as a Ruby class even if you haven't ever set up multisearch.

I've long wanted to get rid of PgSearch::Document and replace it with a generator that generates a class directly in your project. Doing so would prevent this problem. Sorry for the inconvenience.

**Comment by chrisnicola** [comment]
Any workaround I can try for now.

**Comment by chrisnicola** [comment]
Any updates on this, this creates a lot of friction in my debuggingn workflow.

**Comment by nertzy** [comment]
Maybe we should dynamically `require "pg_search/document"` the first time someone calls `multisearchable` instead of always.

**Comment by nertzy** [comment]
Can someone try out this fix and let me know if it works? I want some real-world testing before publishing the gem.

You can try this in your Gemfile:

```ruby
gem 'pg_search', git: 'https://github.com/Casecommons/pg_search.git', ref: 'ebe5385f7aaf223a7aacfb71664ab541acc6e415'
```

**Comment by nertzy** [comment]
Released in v2.3.2
